test/go/src/bin/testequals/main.go

```diff
@@ -0,0 +1,179 @@
+package main
```

Comment:
hmm why do we need to add this directory/file?

Reply:
sry, wrong push